src/reaper_osara.cpp

```diff
@@ -4810,6 +4811,62 @@ void cmdInsertMarker(Command* command) {
 	outputMessage(format(translate("marker {} inserted"), number));
 }
 
+void cmdInsertProjectMarker(Command* command) {
+	cmdhInsertProjectMarker(command->gaccel.accel.cmd);
+}
+
+void cmdhInsertTakeMarker(int command) {
+	if(CountSelectedMediaItems(0)>0) {
+		vector<int> preMarkers(CountSelectedMediaItems(nullptr));
```

Comment:
Do we need the vector here? Can we just add up all the take markers across all selected items now and then after and compare them, rather than comparing the counts for specific items? Am I missing something?

Reply:
I've removed the vector, you're right that it wasn't needed with this approach. I went in more on checking for selected items alongside fakeFocus because this approach keeps the project markers feeling like the primary function of the keystroke, important because those have higher adoption.